src/EmailAccountRecoveryNew.sol

```diff
@@ -145,6 +147,8 @@ abstract contract EmailAccountRecoveryNew {
     /// the subject in the given email auth message.
     function handleAcceptance(
         address account,
+        address validatorModule, // idk if this is even possible or if there is a contraint that
```

Comment:
is this even possible with you current design? im not quite sure if I understand where this function is invoked. So maybe this is a dumb idea.